code/__DEFINES/robots.dm

```diff
@@ -284,6 +299,9 @@ DEFINE_BITFIELD(janitor_mode_flags, list(
 #define FIREBOT_VOICED_ONLY_YOU "Only you can prevent station fires."
 #define FIREBOT_VOICED_TEMPERATURE_NOMINAL "Temperature nominal."
 #define FIREBOT_VOICED_KEEP_COOL "Keep it cool."
+#define FIREBOT_VOICED_CANDLE_TIP "Keep candles near curtains for cozy night lights!"
+#define FIREBOT_VOICED_ELECTRIC_FIRE "Keep full buckets of water near outlets incase of an electric fire!"
+#define FIREBOT_VOICED_GASOLINE_TIP "Pouring gasoline on fire makes it burn out faster!"
```

Comment:
No one probably uses gasoline anymore, should make it `liquid plasma` instead

Reply:
would "fuel" be good enough?

Reply:
i think fuel is nice